app/components/emoji_picker/sections/icons_bar/index.tsx

```diff
@@ -51,8 +51,8 @@ const EmojiSectionBar = ({currentIndex, sections, scrollToIndex}: Props) => {
     return (
         <KeyboardTrackingView
             scrollViewNativeID={SCROLLVIEW_NATIVE_ID}
-            normalList={true}
             style={styles.container}
+            inverted={false}
```

Comment:
Not sure if I missed something typing keyboardtracking view, or this was wrong to begin with. Thoughts?

Reply:
we don't use the inverted option even if the scrollview in itself is inverted.

Reply:
The preview probably lost some context. There are two changes here. Adding `inverted` (which is required, but since it is a boolean, can be set as false without any problem in theory), and removing `normalList`, which I did not find while typing the KeyboardTrackingView component. That is the part were something might be missing.